config/locales/sign_up/react.en.yml

```
@@ -4,16 +4,16 @@ en:
       your_footprint_result_text:
         text_part_before: Your annual carbon footprint is
         text_part_middle: CO2e. Paying
-        text_part_end: will allow you to offset your footprint. The sum is distributed to various climate projects that make an impact for the climate.
+        text_part_end: ' will allow you to offset your footprint. The sum is distributed to various climate projects that make an impact for the climate.'
     category_page:
       category_chart:
         title: Your climate footprint by category
         desc: This is your carbon footprint split per category. Does it match your expectations?
-      public_emissions: Public emissions are the emissions caused by society, depending on where you live. This includes for example government agencies, health care, education and public transport.
+      public_emissions: Public emissions are the emissions caused by the society where you live. Examples of this are government agencies, health care, education and public transport.
```

Insåg att detta kanske ska vara "these"? Eftersom emissions är i plural
```suggestion
      public_emissions: Public emissions are the emissions caused by the society where you live. Examples of these are government agencies, health care, education and public transport.
```

---

Oklart vad vi syftar på. Syftar this/these på "emissions" eller "society" eller något annat?
Jag får inte helt grepp om meningen